### DISCUSSION TITLE
How to get the YOLO achievement?

### DISCUSSION THREAD

**Discussion by undecane** [zingimmick/blog#54] [Q&A] [answered]
Any idea on how to unlock the `YOLO` achievement? I tried merging a pull request without a review as the achievement description says but didn't work.

**Comment by zingimmick** [comment, marked as answer]
1. Create a PR on your own repository
2. Merge it without any checks and a review. If you have any checks turned on, turn them off first. (i.e. https://github.com/zingimmick/.github/pull/58)
3. Done!
